package.json

```diff
@@ -44,7 +47,8 @@
     "build": "npm run build-bundle && npm run build-mangle",
     "test-api": "node test",
     "test-coverage": "nyc --reporter lcov tape test.js",
-    "test": "npm run format && npm run build && npm run test-coverage"
+    "test": "npm run format && npm run build && npm run test-coverage && npm run typecheck",
+    "typecheck": "tsc"
```

Comment:
Could we do additional type checking with dtslint?
for example https://github.com/Rokt33r/hast-util-to-html/blob/318112f2c59275c59d8b687f75f4c21b12486f44/types/tslint.json
https://github.com/syntax-tree/hast-util-to-html/blob/0fef5aa542e1d8a9361f3a0ea8e38adf07fa870f/package.json#L60-L61

Reply:
yes!